.github/workflows/akobot.yml

```diff
@@ -0,0 +1,23 @@
+name: AKOBot
+
+on:
+  schedule:
+    - cron: "0 7 * * 3,5" # At 7:00 on Wednesday and Friday
```

Comment:
Why a cron job?

Reply:
Because we will probably need it to run after we merged the `dependabot` PRs. And some other day of the week for good measure.